### PR TITLE
Flatfile names

### PR DESCRIPTION
**Pull request by brmscheiner**
Individual found:
image

Individual not found:
image

### CODE REVIEW

File: src/pages/bulkImport/BulkReportForm.jsx [hunk 1]
```diff
@@ -13,49 +13,40 @@ import queryKeys from '../../constants/queryKeys';
...
-      }
-    }
-  });
+async function onRecordChange(record, recordIndex) {
+  const messages = validateMinMax(record);
 
-  return recordHookResponse;
+  const individual = record?.individual;
+  if (individual) {
+    const validationResponse = await validateIndividualNames([
```
Comment (Emily-Ke) [inline src/pages/bulkImport/BulkReportForm.jsx:33]: This could throw an error.

**Review by Emily-Ke — CHANGES_REQUESTED** [review]
I know that the minmax values were just copied, but I think some of their values may be incorrect.

File: src/pages/bulkImport/utils/validateMinMax.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+const minmax = {
+  decimalLatitude: [-180, 180],
+  decimalLongitude: [-180, 180],
+  timeYear: [1900, 2021],
```
[the remark this answers is not in the document]
Comment (brmscheiner) [reply src/pages/bulkImport/utils/validateMinMax.js:4]: oh good call. let's just make it 2099, that should hold us over for a while :)

**Review by Emily-Ke — APPROVED** [review]

**Review by Atticus29 — APPROVED** [review]
LGTM

**Pull request merged**